src/main/java/com/systemmeltdown/meltdownlog/lib/LogEntryWriter.java

```diff
@@ -0,0 +1,12 @@
+package com.systemmeltdown.meltdownlog.lib;
+
+@FunctionalInterface
+public interface LogEntryWriter {
+	/**
+	 * Enqueue an entry to the logger.
+	 * @param topicName The name of the topic to log.
+	 * @param value The value to be written. Valid types are String, Integer, Double, Boolean
```

Comment:
Would it be better to have four overloaded writeEntry functions instead of making this a functional interface? Maybe even make this an abstract class where the String method is abstract and the others convert the value to string and forward to it. I think most logging outputs will just want a string value.

Reply:
This system needs to support logging numeric values for graphing over time, also different outputs will output different objects in different ways. They won't always be string output.